packages/components/src/components/as-toolbar/as-toolbar.e2e.ts

```diff
@@ -39,9 +39,12 @@ describe('as-toolbar', () => {
       page.click('.as-toolbar__toggle');
       await page.waitForChanges();
 
-      const actual = await page.find('.as-toolbar__actions');
+      const actual = await page.waitForSelector('.as-toolbar__actions--visible', {
+        timeout: 500,
+        visible: true
+      });
 
-      expect(actual).toHaveClass('as-toolbar__actions--visible');
+      expect(actual).toBeDefined();
```

Comment:
Does this test fails when not clicking on .as-toolbar__toggle ?

Reply:
Yes. I've checked that it fails when you don't click on the toggle or when `as-toolbar__actions--visible` is not available.

Reply:
what does the method return? can we use `.toBeThruty` instead?

Reply:
That was my first intuition, using `.toBeTruthy` but I checked that `toBeDefined` has the desired outcome.

According to the Puppeteer API, `waitForSelector` returns an element if it's found. If it's not found, it throws an error. So this `expect` is there only to have a green check.

I'm refactoring it to reflect better the API and expect no errors (that it's the best thing to do).

Reply:
It's not straightforward to check `not.toThrow` and it leads to false positives.

Just changed to `.not.toBeNull` and checked all the combinations.